GView/src/GView.cpp

```diff
@@ -16,13 +16,26 @@ struct CommandInfo
     std::string_view name;
 };
 
+struct CommandInfoU16
+{
+    CommandID ID;
+    std::u16string_view name;
+};
+
 CommandInfo commands[] = {
     { CommandID::Help, "help" },
     { CommandID::Open, "open" },
     { CommandID::Reset, "reset" },
     { CommandID::UpdateConfig, "updateconfig" },
 };
 
+CommandInfoU16 commandsU16[] = {
```

Comment:
As face un MACRO care este fie unicode fie text si l-as folosi pe ala.
In felul asta definesc o singura data structura de comenzi.